src/_pytest/terminal.py

```diff
@@ -216,6 +216,15 @@ def pytest_addoption(parser: Parser) -> None:
         choices=["auto", "long", "short", "no", "line", "native"],
         help="Traceback print mode (auto/long/short/line/native/no)",
     )
+    group._addoption(
+        "--xfail-tb",
+        metavar="style",
+        action="store",
+        dest="xfail_tbstyle",
+        default="auto",
```

Comment:
I think we should use `no` as default in this case, to get the pre-8.1.0 behavior, as people has been side tracked by this a lot (including myself at work, where we have some xfail in places).

```suggestion
        default="no",
```

We will need to mention this decision very clearly in the changelog.